src/main/src/main/java/org/geoserver/catalog/FeatureTypeInfo.java

```diff
@@ -30,6 +30,9 @@
     /** The cascaded stored query configuration */
     static final String STORED_QUERY_CONFIGURATION = "WFS_NG_STORED_QUERY_CONFIGURATION";
 
+    /** ONLY FOR WFS-NG Feature Types */
```

Comment:
If it is only for ``WFS-NG Feature Types`` (not sue what does this means) the it shoudl not be here.

Reply:
this is key that will be used to store other supported SRS for a remote Feature type coming from WFS capability document. If the GetMap or GetFeature request have spatial filter with SRS belonging to one of the stored SRS, then the changes of this PR will ensure that they are passed down to the query level and no un-necessary re-projection occurs.